File: dataset/human/java/file_2547.java
import java.util.*;
public class Main{
	public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        char[] s = sc.next().toCharArray();
        char[] t = sc.next().toCharArray();
        Arrays.sort(s);
        Arrays.sort(t);
        String sort_s = String.valueOf(s);
        String sort_t = String.valueOf(t);

        String sort_rev_t = new StringBuilder(sort_t).reverse().toString();

        if(sort_s.compareTo(sort_rev_t) < 0){
            System.out.println("Yes");
        }else{
            System.out.println("No");
        }

    }
    
}

File: dataset/human/c/file_2547.c
#include<stdio.h>
int main(){
 int c=1,d=0;
char a[3];
scanf ("%s",a);
  for (int i=0;i<3;i++){
  if (a[i]=='a'){
    d=d+1;
  }
  else if(a[i]=='b'){
  c=c*2;
  d=d+2;
  }
  else if(a[i]=='c'){
    c=c*3;
  d=d+3;
  }
  }
  if(c==6 & d==6){
    printf("Yes");
  }
  else{
  printf ("No");
  }
   return 0;
}
